integrated_channels/sap_success_factors/exporters/content_metadata.py

```diff
@@ -168,6 +169,10 @@ def transform_schedule(self, content_metadata_item):
         """
         Return the schedule of the content item.
         """
+        if content_metadata_item.get('content_type') == 'course':
+            course_run = get_advertised_or_closest_course_run(content_metadata_item)
```

Comment:
here as well for `normalized_metadata`

Reply:
Makese sense but in this case, we also need to check whether the course run is available for enrollment. For that, we need extra information that is not available in `normalized_metadata` but is available in the course run body.